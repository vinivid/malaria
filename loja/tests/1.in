I,Livro,8535283528,Introdução ao teste de software,Marcio Delamaro,GEN LTC,2017,2,448,PT-BR
I,Livro,8535226346,Introdução ao teste de software,Marcio Delamaro,ELSEVIER,2005,1,394,PT-BR
I,Livro,8522104999,Introdução á teoria da computação,Michael Sipser,Cengage Learning,2005,1,612,PT-BR
I,CD,8579715172,COLECAO MILTON NASCIMENTO,Milton Nascimento,10,Abril,2022
I,CD,093624684725,Ray Of Light,Madona,6,Rimo,2010
I,DVD,7899307921551,Tinker Bell E O Monstro Da Terra Do Nunca,John Doe,PT-BR,Animação,2003,Estadunidense
A,8535226346,10
A,8535283528,66
A,8535226346,33
P,8522104999
A,8522104999,22
A,8579715172,0
P,093624684725
A,093624684725,10
A,7899307921551,15
V,8579715172,1
A,8535226346,11
A,1234567890,2
V,1234567789,2
V,8535226346,11
P,1234567890
P,Introdução ao teste de software
V,093624684725,10
S
I,DVD,7899307921551,Tinker Bell E O Monstro Da Terra Do Nunca,John Doe,PT-BR,Animação,2003,Estadunidense
