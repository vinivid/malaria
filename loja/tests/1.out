
Operação inserir livro: 8535283528
Operação realizada com sucesso

Operação inserir livro: 8535226346
Operação realizada com sucesso

Operação inserir livro: 8522104999
Operação realizada com sucesso

Operação inserir CD: 8579715172
Operação realizada com sucesso

Operação inserir CD: 93624684725
Operação realizada com sucesso

Operação inserir DVD: 7899307921551
Operação realizada com sucesso

Operação de compra: 8535226346
Operação realizada com sucesso: 8535226346

Operação de compra: 8535283528
Operação realizada com sucesso: 8535283528

Operação de compra: 8535226346
Operação realizada com sucesso: 8535226346

Procurando: 8522104999
Encontrado:
Livro
Código: 8522104999
Título: Introdução á teoria da computação
Autor: Michael Sipser
Editora: Cengage Learning
Edição: 1
Ano: 2005
Páginas: 612
Idioma: PT-BR

Operação de compra: 8522104999
Operação realizada com sucesso: 8522104999

Operação de compra: 8579715172
Operação realizada com sucesso: 8579715172

Procurando: 093624684725
Encontrado:
CD
Código: 93624684725
Título: Ray Of Light
Banda: Madona
Gravadora: Rimo
Ano: 2010
trilhas: 6

Operação de compra: 93624684725
Operação realizada com sucesso: 93624684725

Operação de compra: 7899307921551
Operação realizada com sucesso: 7899307921551

Operação de venda: 8579715172
***Erro: Estoque insuficiente: 8579715172 Quantidade: 1

Operação de compra: 8535226346
Operação realizada com sucesso: 8535226346

Operação de compra: 1234567890
***Erro: Código inexistente: 1234567890

Operação de venda: 1234567789
***Erro: Código inexistente: 1234567789

Operação de venda: 8535226346
Operação realizada com sucesso: 8535226346

Procurando: 1234567890
Produto não encontrado: 1234567890

Procurando: Introdução ao teste de software
Encontrado:
Livro
Código: 8535283528
Título: Introdução ao teste de software
Autor: Marcio Delamaro
Editora: GEN LTC
Edição: 2
Ano: 2017
Páginas: 448
Idioma: PT-BR

Operação de venda: 93624684725
Operação realizada com sucesso: 93624684725

Operação de sumarização: 
Livro
Código: 8535283528
Título: Introdução ao teste de software
Autor: Marcio Delamaro
Editora: GEN LTC
Edição: 2
Ano: 2017
Páginas: 448
Idioma: PT-BR
Quantidade: 66

Livro
Código: 8535226346
Título: Introdução ao teste de software
Autor: Marcio Delamaro
Editora: ELSEVIER
Edição: 1
Ano: 2005
Páginas: 394
Idioma: PT-BR
Quantidade: 43

Livro
Código: 8522104999
Título: Introdução á teoria da computação
Autor: Michael Sipser
Editora: Cengage Learning
Edição: 1
Ano: 2005
Páginas: 612
Idioma: PT-BR
Quantidade: 22

Quantidade de Livros: 131

CD
Código: 8579715172
Título: COLECAO MILTON NASCIMENTO
Banda: Milton Nascimento
Gravadora: Abril
Ano: 2022
trilhas: 10
Quantidade: 0

CD
Código: 93624684725
Título: Ray Of Light
Banda: Madona
Gravadora: Rimo
Ano: 2010
trilhas: 6
Quantidade: 0

Quantidade de CDs: 0

DVD
Código: 7899307921551
Título: Tinker Bell E O Monstro Da Terra Do Nunca
Diretor: John Doe
Gênero: Animação
Ano: 2003
Nacionalidade: Estadunidense
Idioma: PT-BR
Quantidade: 15

Quantidade de DVDs: 15


Operação inserir DVD: 7899307921551
***Erro: Código já cadastrado: 7899307921551
